views/js/runner/helpers/map.js

```diff
@@ -302,8 +303,17 @@ define([
             }
 
             // update the stats to reflect the scope
-            if (section) {
-                section.stats = this.computeItemStats(section.items);
+            if (context && context.options && context.options.reviewSkipahead) {
+                partSections = part.sections || {};
+                if (partSections) {
+                    _.each(partSections, function(partSection) {
```

Comment:
`_.each` is depreacted, please use `_.forEach` instead.

Reply:
Hmmm? I can change this, but can you point me on the deprecation info?
Because neither at [Lodash deprecations](https://github.com/lodash/lodash/wiki/Deprecations), nor at [Lodash changelog](https://github.com/lodash/lodash/wiki/Changelog) I see it deprecated. Just only found that `each` is an alias for `forEach`. Quick googling also didn't gave any results for that.

Reply:
it is an alias now, the exact API is `forEach`: https://lodash.com/docs/2.4.2#forEach

So, yes, please don't use the alias as it may disappear in later versions

Reply:
But, anyway, this part of the code should go IMO. The `prevent-unseen` option already does the job. The only thing you have to do is add another condition to allow the disclose of unseen item for a particular category.

Reply:
Aw, yes, understood about using alias, will move it to use the `forEach`.

About `prevents-unseen` - please, see the comment to the last one of yours in here